recipes/hiveplot/meta.yaml

```diff
@@ -0,0 +1,53 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.1.8.4" %}
+{% set pkgname = "hiveplot" %}
+{% set maintainer = "ericmjl" %}
+
+package:
+    name: {{ pkgname }}
+    version: {{ version }}
+
+source:
+    fn: {{ pkgname }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/h/{{ pkgname }}/{{ pkgname }}-{{ version }}.tar.gz
+    # md5 is OK but sha256 is desirable.
+    md5: 08171310271823a83aff8269eeed2392
+
+build:
+    number: 0
+    # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+        - setuptools
+        - matplotlib
+        - numpy
+        - pip
+        - icu
```

Comment:
I think that those three should be sufficient. Please pardon my ignorance, but in what circumstances would anything more than those 3 be necessary for the build stage? Would building Cython extensions be one case, in which Cython would be required then?